src/core_plugins/console/index.js

```diff
@@ -32,11 +32,7 @@ export default function (kibana) {
     require: [ 'elasticsearch' ],
 
     isEnabled(config) {
-      // console must be disabled when tribe mode is configured
-      return (
-        config.get('console.enabled') &&
-        !config.get('elasticsearch.tribe.url')
-      );
+      return config.get('console.enabled');
```

Comment:
I believe this is default behavior if isEnabled is undefined, (as long as console.enabled is defined)

Reply:
I think you're right.  I'll remove this whole block before merging.